web/pdf_print_service.js

```diff
@@ -133,10 +132,11 @@
       this.scratchCanvas = null;
       activeService = null;
       ensureOverlay().then(function () {
-        if (OverlayManager.active !== 'printServiceOverlay') {
+        var overlayManager = PDFViewerApplication.overlayManager;
```

Comment:
Nit: I'm wondering if it might not be a bit cleaner to just have this definition at the top of the file instead (like it currently is, since that'd reduce the total diff).